lib/Mojolicious/Guides/Cookbook.pod

```diff
@@ -342,6 +342,56 @@ web features, such as WebSockets, are not available.
 
   ScriptAlias / /home/sri/my_app/script/my_app/
 
+=head2 Envoy
+
+L<Mojolicious> applications can be deployed on "cloud-native" environments that use L<Envoy|https://www.envoyproxy.io>,
+such as with this reverse proxy configuration similar to the Apache and Nginx ones above.
+
+  static_resources:
+    listeners:
+    - name: listener_0
+      address:
+        socket_address: { address: 0.0.0.0, port_value: 80 }
+      filter_chains:
+      - filters:
+        - name: envoy.filters.network.http_connection_manager
+          typed_config:
+            "@type": type.googleapis.com/envoy.extensions.filters.network.http_connection_manager.v3.HttpConnectionManager
+            codec_type: auto
+            stat_prefix: index_http
+            route_config:
+              name: local_route
+              virtual_hosts:
+              - name: service
+                domains: ["*"]
+                routes:
+                - match:
+                    prefix: "/"
+                  route:
+                    cluster: local_service
+            upgrade_configs:
+            - upgrade_type: websocket
+            http_filters:
+            - name: envoy.filters.http.router
+              typed_config:
+    clusters:
+    - name: local_service
+      connect_timeout: 0.25s
+      type: strict_dns
+      lb_policy: round_robin
+      load_assignment:
+        cluster_name: local_service
+        endpoints:
+        - lb_endpoints:
+          - endpoint:
+              address:
+                socket_address: { address: mojo, port_value: 8080 }
+                
+While this configuration works for simple applications, Envoy's typical use case is for implementing proxies of
+applications as a "service mesh" providing advanced filtering, load balancing, and observability features, such as
+seen in L<Istio|https://istio.io/latest/docs/ops/deployment/architecture/>.  More examples can be found at the
```

Comment:
Extra whitespace.

Reply:
Is "at the documentation" is correct english?

Reply:
Would you prefer
```suggestion
seen in L<Istio|https://istio.io/latest/docs/ops/deployment/architecture/>. For more examples, see the
```

Reply:
I'll use the above suggestion then, we can revisit it in the new review pass :muscle: